packages/odyssey-design-tokens/src/color/functional.json

```diff
@@ -0,0 +1,28 @@
+{
+  "color": {
```

Comment:
leaving a subsection off here purposefully. Although it runs against style-dictionary norms (no big deal really) it aligns with what @arnoldsandoval-okta setup here https://docs.google.com/document/d/1ZXQahqi_ihNnHIGqE_VoruOzt5CVWmgnJq2_niHI51w/edit.

as an example, we want these colors to output as `ColorPrimaryLight` and not something like `ColoBasePrimaryLight`. The `Base` being a corresponding grouping under color here similar to `Palette` which **does** exist